geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/serial/SerialWANPersistenceEnabledGatewaySenderDUnitTest.java

```diff
@@ -577,17 +577,15 @@ public void testReplicatedRegionPersistentWanGateway_restartSenderWithCleanQueue
     Integer nyPort = (Integer) vm1.invoke(() -> WANTestBase.createFirstRemoteLocator(2, lnPort));
 
     createCacheInVMs(nyPort, vm2, vm3);
-    createReceiverInVMs(vm2, vm3);
 
     createCacheInVMs(lnPort, vm4, vm5, vm6, vm7);
 
-    String firstDStore = (String) vm4.invoke(() -> WANTestBase.createSenderWithDiskStore("ln", 2,
-        false, 100, 10, false, true, null, null, true));
-    String secondDStore = (String) vm5.invoke(() -> WANTestBase.createSenderWithDiskStore("ln", 2,
-        false, 100, 10, false, true, null, null, true));
-
-    logger.info("The first ds is " + firstDStore);
-    logger.info("The second ds is " + secondDStore);
+    String firstDStore =
+        (String) vm4.invoke("Creating DS", () -> WANTestBase.createSenderWithDiskStore("ln", 2,
+            false, 100, 10, false, true, null, null, true));
+    String secondDStore =
+        (String) vm5.invoke("Creating DS", () -> WANTestBase.createSenderWithDiskStore("ln", 2,
+            false, 100, 10, false, true, null, null, true));
```

Comment:
These variables are never used, so I think it should be okay to change these to just 
```
vm4.invoke("Creating DS", () -> WANTestBase.createSenderWithDiskStore("ln", 2, false, 100, 10, 
false, true, null, null, true));
vm5.invoke("Creating DS", () -> WANTestBase.createSenderWithDiskStore("ln", 2, false, 100, 10, 
false, true, null, null, true));
```